src/AnimationExport/exportAnimation.jl

```diff
@@ -197,6 +197,31 @@ function exportObject(object, elements, obj::Modia3D.Composition.Object3D, coord
     return (r_obj, R_obj)
 end
 
+function exportObject(object, elements, obj::Modia3D.Composition.Object3D, grid::Modia3D.Shapes.Grid, initPos, initRot)
+    r_obj = Modia3D.ZeroVector3D
+    R_obj = Shapes.rotateAxis2z(grid.axis, Modia3D.NullRotation)
+    name = Modia3D.fullName(obj)
+    geometryName = name * ".geometry"
+    array = []
+    for i in 0:div(grid.length+1e-12, grid.distance)
```

Comment:
Is this code line type stable? I guess div(a,b) is a Float, therefore it's better `0.0:div(a,b)`